datamol/mol.py

```diff
@@ -44,6 +48,10 @@
 AROMATIC_BOND = Chem.rdchem.BondType.AROMATIC
 DATIVE_BOND = Chem.rdchem.BondType.DATIVE
 UNSPECIFIED_BOND = Chem.rdchem.BondType.UNSPECIFIED
+SALT_PATH = str(importlib_resources.files("datamol").joinpath("data/salts.smi"))
+SALT_REMOVER = SaltRemover(defnFilename=SALT_PATH)
+SOLVENT_PATH = str(importlib_resources.files("datamol").joinpath("data/solvents.smi"))
```

Comment:
Maybe you should use `dm.data.open_datamol_data_file()` to load the files since it will take into account the new API that are different in between 3. and 3.10.

Function at https://github.com/datamol-io/datamol/blob/9bea9401c8d41fda4250dc92564c595ebaf27718/datamol/data/__init__.py#L18-L54

In that case you can remove the `importlib_resources` import from above.